tests/unit/ssg-module/test_templates.py

```diff
@@ -13,6 +13,7 @@
 ssg_root = os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
 DATADIR = os.path.abspath(os.path.join(os.path.dirname(__file__), "data"))
 templates_dir = os.path.join(DATADIR, "templates")
+platforms_dir = os.path.join(DATADIR)
```

Comment:
You don't have to join if you have only a single item.

Reply:
Fixed.